src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,108 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    public void register_existingLogin_notOk() {
+        User user = new User();
+        user.setLogin("Good login");
+        user.setPassword("Good password");
+        user.setAge(19);
+
+        Storage.people.add(user);
+
+        User newUser = new User();
+        newUser.setLogin("Good login");
+        newUser.setPassword("Good password");
+        newUser.setAge(19);
+
+        assertThrows(RegistrationException.class,
+                () -> registrationService.register(newUser),
+                "Expected register() to throw, but it didn't");
+    }
+
+    @Test
+    public void register_allNull_notOk() {
+        User user = new User();
+        user.setLogin(null);

```

Comment:
there is no sense to make all fields null, when you call register method first check login == null will throw an exception and other fields will be ignored
you need to create 3 test cases login, password, age are null separately